### ISSUE TITLE
view Tomte VCF in Scout

### ISSUE THREAD

**Issue by Lucpen** [Clinical-Genomics/scout#4700]
**Is your feature request related to a problem in the current program to new available techology or software? Please describe and add links/citations if appropriate.**
Would it be possible to view the vcf output of tomte in Scout, there is information there about SNPs and VAF that could potentially be interesting

**Describe the solution you'd like**
To see vcf in Scout

**Describe alternatives you've considered**

**Additional context**

**Comment by dnil** [comment]
Are there any particular keys missing that you don't get if you do that today?

Overall, this is probably more of a policy question. In principle, it is already be possible to load a WTS SNV VCF on the case, assuming there is no WGS SNV VCF, which we would typically wish there is. The original intent for Tomte or a follow up to it afaik was to annotate the WGS with info from the WTS calls, e.g. with AF imbalance and potential allele specific expression calls. I even understood there is a drop module for this, if we can just provide the SNV vcf as input?

**Comment by Lucpen** [comment]
Yes, I was more thinking on the Allelic imbalance part (there is a field regarding that in the vcf), but they should be able to asses that (at least partially) from the igv view

**Comment by dnil** [comment]
Good, what is the key? I had a quick look through the VCF header, but must have missed it?

**Comment by Lucpen** [comment]
I think this one will have to wait until you guys are back from vacation as they were added in the latest release, which is not what has been run in production. It will probably be added next week, I will add a comment with the header as soon as that happens

**Comment by dnil** [comment]
I see. Have you solved the issue with pulling in genome wgs as well so that we get calls imbalance calls also for completely silent alleles?

**Comment by Lucpen** [comment]
Not yet, that will happen in the future. At the moment we only detect allelic imbalance rather than monoallelic expression. In other words, we just look at how many calls come from each allele if more than one allele is found in RNAseq. Then we use Bootstrapann to calculate significance.

**Comment by dnil** [comment]
It’s a start! 👍🏻

**Comment by dnil** [comment]
Maybe you already have it in the newer version, but I noticed the old one was lacking genmod annotation? As you know, there is an nf-core module for it you could drop in, and get inheritance model support for that VCF as well as a basic scoring. Maybe you would like to incorporate the imbalance prediction in your score model as well?
